index.html

```diff
@@ -40,7 +40,7 @@
       Number <input type="number" name="age" ><br>
```

Comment:
Why have you changed in `index.html` file!? .. Undo the changes

Reply:
change kori nai.. open cilo sob file ..akta jaygay delate hye gesilo .. ami oita abar likhe save kore disi just .

Reply:
hm. You added a whitespace after `type="date` & a new line after 79 # line